dashboard/service.py

```diff
@@ -86,3 +86,16 @@ def start_openid_auth():
 
     return_to = url_for('auth.signed_in', _external=True)
     return oid_request.redirectURL(request.url_root, return_to=return_to)
+
+
+def start_buildset(pipeline, buildset):
```

Comment:
@kam193 Start/Stop services should be a class or classes with the same interface. They take exactly the same arguments and internally they are almost the same. One difference is that Start will call `zuul enqueue` and Stop will call `zuul dequeue`. Internal mechanisms stay the same.

Reply:
I am still waiting for the unification of the vocabulary.